Added 2 extra steps to Reclaiming PostgreSQL Space (#1085)
theforeman/foreman-documentation: guides/common/modules/proc_reclaiming-postgresql-space.adoc

----
# {foreman-maintain} service start
----

> . Confirm files are present under `/var/lib/pgsql/` and check its size.

Reviewer: ```suggestion
. Confirm that the files exist in the `/var/lib/pgsql/` directory:
```
This is two steps. Add a separate step for checking the size. One command per step.

+
[options="nowrap" subs="+quotes,attributes"]
----
# ls -l /var/lib/pgsql/

# du -sh /var/lib/pgsq/
----

> . If you are using {ProjectServer} {ProjectVersionPrevious} and higher, the data from `/var/lib/pgsql/` can be deleted.

Reviewer: We only document for the specific version in the specific version of the docs. 
```suggestion
. Delete the data from the `/var/lib/pgsql/` directory: 
```

Author: @melcorr I made the requested changes and made another commit. Let me know if there are any issues seeing my commit because my VPN went down when I initially submitted the commit.

Reviewer: Hey @bangelic can you do 

git log 
see can you see your second commit. 
Then do git push origin. It isn't here yet :-)
